feature-detects/es5/strictmode.js

```diff
@@ -2,10 +2,10 @@
 {
   "name": "ES5 Strict Mode",
   "property": "strictmode",
-  "caniuse": "sctrict-mode",
```

Comment:
lol

Reply:
by the way: I cant find a usage of the caniuse property, is that a TODO or where is it used?